diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6c667df..8ebae74 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,24 +104,6 @@ if (NOT CPPKAFKA_PKGCONFIG_DIR)
     set(CPPKAFKA_PKGCONFIG_DIR share/pkgconfig)
 endif()
 
-# Look for Boost (just need boost.optional headers here)
-find_package(Boost REQUIRED ${FIND_PACKAGE_QUIET})
-
-if (Boost_FOUND)
-    find_package(Boost COMPONENTS program_options ${FIND_PACKAGE_QUIET})
-    set(Boost_USE_STATIC_LIBS ${CPPKAFKA_BOOST_STATIC_LIBS})
-    set(Boost_USE_MULTITHREADED ${CPPKAFKA_BOOST_USE_MULTITHREADED})
-    include_directories(${Boost_INCLUDE_DIRS})
-    link_directories(${Boost_LIBRARY_DIRS})
-    if (CPPKAFKA_CMAKE_VERBOSE)
-        message(STATUS "Boost include dir: ${Boost_INCLUDE_DIRS}")
-        message(STATUS "Boost library dir: ${Boost_LIBRARY_DIRS}")
-        message(STATUS "Boost use static libs: ${Boost_USE_STATIC_LIBS}")
-        message(STATUS "Boost is multi-threaded: ${CPPKAFKA_BOOST_USE_MULTITHREADED}")
-        message(STATUS "Boost libraries: ${Boost_LIBRARIES}")
-    endif()
-endif()
-
 # Try to find the RdKafka configuration file if present.
 # This will search default system locations as well as RdKafka_ROOT and RdKafka_Dir paths if specified.
 find_package(RdKafka ${FIND_PACKAGE_QUIET} CONFIG)
diff --git a/cmake/config.cmake.in b/cmake/config.cmake.in
index 32f4398..749790c 100644
--- a/cmake/config.cmake.in
+++ b/cmake/config.cmake.in
@@ -8,7 +8,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}")
 set(RDKAFKA_MIN_VERSION_HEX "@RDKAFKA_MIN_VERSION_HEX@")
 
 # Find boost optional
-find_dependency(Boost REQUIRED)
+# find_dependency(Boost REQUIRED)
 
 # Try to find the RdKafka configuration file if present.
 # This will search default system locations as well as RdKafka_ROOT and RdKafka_DIR paths if specified.
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 214ccd4..7d73ad6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -51,7 +51,7 @@ set_target_properties(${TARGET_NAME} PROPERTIES
         VERSION ${CPPKAFKA_VERSION}
         SOVERSION ${CPPKAFKA_VERSION})
 # In CMake >= 3.15 Boost::boost == Boost::headers
-target_link_libraries(${TARGET_NAME} PUBLIC RdKafka::rdkafka Boost::boost)
+target_link_libraries(${TARGET_NAME} PUBLIC RdKafka::rdkafka)
 if (WIN32)
     # On windows ntohs and related are in ws2_32
     target_link_libraries(${TARGET_NAME} PUBLIC ws2_32.lib)
